src/neuroconv/datainterfaces/text/timeintervalsinterface.py

```diff
@@ -0,0 +1,63 @@
+from abc import abstractmethod
+
+import numpy as np
+from pynwb import NWBFile
+from pynwb.epoch import TimeIntervals
+
+from ...basedatainterface import BaseDataInterface
+from ...utils.types import FilePathType, Optional
+from ...tools.nwb_helpers import make_or_load_nwbfile
+
+
+def convert_df_to_time_intervals(df, name, description=None):
+    if description is None:
+        description = name
+    time_intervals = TimeIntervals(name, description)
+    if "start_time" not in df:
+        raise ValueError(f"df must contain a column named 'start_time'. Existing columns: {df.columns.to_list()}")
+    if "stop_time" not in df:
+        df["stop_time"] = np.nan
```

Comment:
So, this is a design question - do we want to enforce/require people to modify the column names in whatever file format they're using to conform with NWB column names?

Or, I was originally thinking one of the arguments to an interface like this could be `column_mapping` or `column_name_mapping` or anything better. Essentially a `Dict[str, str]` mapping the names from their lab-specific CSV format to how they want the names to be in NWB format (which we can enforce best practices like `_time` on). That way they don't have to comb through potentially hundreds of their own files changing things from how they're used to doing it and can instead just tell us what the mapping is to have it done seamlessly

Reply:
good idea. I'll add that

Reply:
done